ipset.go

```diff
@@ -378,5 +424,16 @@ func (s *IPSet) RemoveFreePrefix(bitLen uint8) (p IPPrefix, newSet *IPSet, ok bo
 	var b IPSetBuilder
 	b.AddSet(s)
 	b.RemovePrefix(prefix)
-	return prefix, b.IPSet(), true
+	newSet, _ = b.IPSet()
```

Comment:
I'd feel better about a panic on error here as that means some fundamental invariants have been violated.